### PR TITLE
Add various talks about JUnit 5

### PR DESCRIPTION
**Pull request by nicolaiparlog**
I included my talks.

---

I hereby agree to the terms of the JUnit Contributor License Agreement.

### CODE REVIEW

**Comment by nicolaiparlog** [comment]
I saw that there seems to be confusion about how exactly the `title` is supposed to be defined. Wouldn't it be better to have `title` and `location` separated so that they can be put together by Jekyll? This would guarantee uniformity.

**Comment by sbrannen** [comment]
Wow... you're gonna (continue to) be quite active on the speaker circuit. Congrats! And thanks for spreading the word!

**Comment by sbrannen** [comment]
Regarding the title, I was just going to say...

Please include the title of your talk and not just the name of the event/conference.

**Comment by sbrannen** [comment]
> I saw that there seems to be confusion about how exactly the `title` is supposed to be defined. Wouldn't it be better to have `title` and `location` separated so that they can be put together by Jekyll? This would guarantee uniformity.

I agree.

Though I propose `event` instead of `location.

Would you mind opening a new issue to address that?

TIA

File: data/events.json [hunk 1]
```diff
@@ -148,6 +148,24 @@
     ]
   },
   {
+    "title": "Java User Group Mannheim",
```
Comment (sbrannen) [inline data/events.json]: Please include the title of your talk and not just the name of the event/conference.

Do something similar to the other entries that use a hyphen to separate the two, and please do that for all of talks.

Cheers

Comment (sbrannen) [inline data/events.json]: Please include the title of your talk and not just the name of the event/conference.

Do something similar to the other entries that use a hyphen to separate the two, and please do that for all of talks.

Cheers

**Comment by nicolaiparlog** [comment]
> And thanks for spreading the word!

It's not like I'm not also proposing other topics - this is just the one program committees apparently like the most. :wink:

**Comment by marcphilipp** [comment]
Thanks, @nicolaiparlog! Do you have time to add the `event` property and adapt the Javascript accordingly? If not, please open an issue for it. 🙂

**Comment by nicolaiparlog** [comment]
Just opened #611 for that. I will have a look tonight to see whether I can make it work. I made a quick repo search for "title" which came up empty - that confused me. I will spend more brain power on that later.

**Comment by sbrannen** [comment]
@nicolaiparlog it's in this branch: https://github.com/junit-team/junit5/tree/gh-pages

**Comment by sbrannen** [comment]
... but you obviously know about that branch, so now _I'm_ confused. 😜

**Comment by nicolaiparlog** [comment]
Mmh, [turns out](https://help.github.com/articles/searching-code/) GitHub's search only looks through the repo's default branch. Had no problem to find it on disk. PR incoming...

**Pull request merged**